hydrogym/firedrake/envs/cylinder/flow.py

```diff
@@ -242,7 +242,7 @@ def cyl_velocity_field(self):
     theta_lo = -0.5 * pi
     A_lo = ufl.conditional(
         abs(theta - theta_lo) < omega / 2,
-        pi / (2 * omega * self.rad**2) * ufl.cos(
+        -pi / (2 * omega * self.rad**2) * ufl.cos(
```

Comment:
Critical fix that has not been considered in https://github.com/dynamicslab/hydrogym/pull/195 ?

Reply:
I don't know, did I get this wrong?  I think this is separate from the fixes in #195

Reply:
Hello @jcallaham , if you have a moment to sort this out I would be very grateful since I am writing my PhD thesis and I am about to launch experiments.
Thank you:)

Reply:
thank you very much @jcallaham :)!